styles/main.scss

```diff
@@ -1,9 +1,44 @@
-$color: blue;
+#rajoyirize-container {
+    font-family: Arial;
+    font-size: 16px;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    overflow: hidden;
+    
+    &.hidden {
+        display: none !important;
+    }
+    
+    .rajoyirize-content-wrapper {
```

Comment:
That content wrapper allows to animate all elements (image and quotes) as a whole element, having the animation area delimited by #rajoyirize-container.